cores/esp8266/core_esp8266_si2c.cpp

```diff
@@ -112,7 +112,7 @@ class Twi
     bool _slaveEnabled = false;
 
     // Internal use functions
-    void ICACHE_RAM_ATTR busywait(unsigned char v);
+    void ICACHE_RAM_ATTR busywait(unsigned int v);
```

Comment:
Comment of intent:
Strictly speaking, this is a signature change, and as such it could potentially be a breaking change.
However, anyone using the (very new and supposedly internal) Twi class directly deserves to get yelled at, so let's keep it as proposed.

Reply:
busywait()  is a private function,  so it shouldn't change the signature